recipe/meta.yaml

```diff
@@ -15,13 +15,14 @@ source:
     - 12.10-fixes.patch
 
 build:
-  number: 2
+  number: 3
   skip: true  # [win or py3k]
 
 requirements:
   build:
+    - python 2.7
```

Comment:
Remove this. `python` is in `host`

Reply:
OK

Reply:
On my laptop, I am using anaconda3, but with a python 2.7 environment. But I had to run

    conda build --python=2.7 recipe

in order to get it to build (I skip the `py3k` build, and the build was being skipped). Will moving `python` to `host` fix it so I don't have to specify `--python=2.7`?

Reply:
You should do `conda build recipe -m .ci_support/linux_python2.7.yaml`.

Reply:
Re: output multiple packages, this would be good because it would be easier to keep the two in sync.